packages/core/src/render3/error_code.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export const enum RuntimeErrorCode {
+  // Internal Errors
+
+  // Change Detection Errors
+  EXPRESSION_CHANGED_AFTER_CHECKED = '100',
+
+  // Dependency Injection Errors
+  INSTANTIATE_CYCLIC_DEPENDENCY = '200',
+  NO_PROVIDER = '201',
```

Comment:
Just a quick comment: I'd propose the following update for consistency with other codes:
```suggestion
  PROVIDER_NOT_FOUND = '201',
```